src/main/scala/nl/knaw/dans/easy/multideposit/CommandLineOptions.scala

```diff
@@ -56,7 +54,7 @@ object CommandLineOptions {
         LdapImpl(new InitialLdapContext(env, null))
       })
 
-    log.debug(s"Using the following settings: $settings")
+    debug(s"Using the following settings: $settings")
```

Comment:
I see what you mean. `$settings` calls the `toString`, which only prints `multidepositDir`, `springfieldInbox` and `outputDepositDir`. So, no passwords from LDAP are printed!